### PR TITLE
chore: Test scalameta changes

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala [hunk 1]
```diff
@@ -1151,17 +1151,17 @@ object MetalsEnrichments
...
         case _ => Iterator.empty
       }
 
     def trailingTokens: Iterator[m.Token] =
       tree.origin match {
-        case Origin.Parsed(input, dialect, pos) =>
-          val tokens = dialect(input).tokenize.get
-          tokens.slice(pos.end, tokens.length).iterator
+        case parsed: Origin.Parsed =>
+          val tokens = parsed.dialect(parsed.input).tokenize.get
```
Comment (kitbellew) [inline metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala:1163]: parsed.source.tokens

File: metals/src/main/scala/scala/meta/internal/parsing/Trees.scala [hunk 2]
```diff
@@ -32,6 +33,7 @@ final class Trees(
 )(implicit reports: ReportContext) {
 
   private val trees = TrieMap.empty[AbsolutePath, Tree]
+  private val tokens = TrieMap.empty[AbsolutePath, Tokens]
```
Comment (kitbellew) [inline metals/src/main/scala/scala/meta/internal/parsing/Trees.scala:36]: is this used?

**Review by kitbellew — COMMENTED** [review]
some test failures are due to sorting of imports. others i don't understand.

File: metals/src/main/scala/scala/meta/internal/parsing/Trees.scala (hunk 2 above)
Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/parsing/Trees.scala:36]: ach, that was from another issue, just forgot to remove it.

**Comment by tgodzik** [comment]
> some test failures are due to sorting of imports. others i don't understand.

Looks like the failures are only related to not handling snapshot well. We should be fine to release.

**Pull request closed without merging**